### PR TITLE
UI/New Car Selection Attributes

### PR DESCRIPTION
**Pull request by DJSteins**
ezgif com-video-to-gif

### CODE REVIEW

File: Assets/_PartyParkade/Script/UI/MainMenu/PlayerSelection/PlayerSelectionVC/CarStats/CarStatSelectionViewController.cs [hunk 1]
```diff
@@ -127,8 +150,8 @@ private void Awake()
 
         private void Start()
         {
-            m_weightBar.localScale = ZeroLength;
-            m_speedBar.localScale = ZeroLength;
+            // RotateStat(m_weightBar, 180f);
+            // RotateStat(m_speedBar, 180f);
```
Comment (Gursaanj) [inline Assets/_PartyParkade/Script/UI/MainMenu/PlayerSelection/PlayerSelectionVC/CarStats/CarStatSelectionViewController.cs:154]: If these are being commented out should we just delete the start method then?

**Comment by mikeslgoh** [comment]
[(1) PlayerSelection: Implement handling attribute](https://trello.com/c/YJbLJP2W/569-1-playerselection-implement-handling-attribute)

**Comment by mikeslgoh** [comment]
[(1) Update player attributes to follow the mockups on Figma](https://trello.com/c/WtU3pO4t/568-1-update-player-attributes-to-follow-the-mockups-on-figma)

**Comment by m4rtint** [comment]
Can you also delete the unused assets  (the original bars png)?

**Comment by m4rtint** [comment]
Can the words and the bars be centered in the middle?

**Pull request merged**